rollup.config.js

```diff
@@ -92,14 +92,6 @@ export default (name, outputFileName, globals, ignoreEsm) => {
                 cleanup()
             ]
         };
-
-        if (name === 'Web3') {
-            config[1].input = 'src/Web3.js';
-        }
-    }
-
-    if (name === 'Web3') {
-        config[0].input = 'src/Web3.js';
```

Comment:
Do we need this line for src/rsk3.js?

Reply:
I have not verified the impact of this change, I assume that this is OK